src/routes/Group/components/TicketView.js

```diff
@@ -23,37 +23,102 @@ class TicketView extends React.Component {
     }
 
     this.state = {
-      showOpenTickets
+      showOpenTickets,
+      activeIndex: 0
     };
     _.bindAll(this,
-      'handleTabClick');
+      'handleTabClick',
+      'handleAccordionTitleClick');
   }
 
   handleTabClick() {
     this.setState({ showOpenTickets: !this.state.showOpenTickets });
   }
 
+  handleAccordionTitleClick(e, i) {
+    this.setState({ activeIndex: this.state.activeIndex === i ? -1 : i })
+  }
+
+  formatDate(date) {
+    return moment(date).format('MMMM D YYYY, h:mm:ss a');
+  }
+
+  eventHelper(index, timestamp, summary) {
+    return (
+      <Feed.Event key={index}>
+        <Feed.Content>
+          <Feed.Date content={String(this.formatDate(timestamp))} />
+          <Feed.Extra>
+            {summary}
+          </Feed.Extra>
+        </Feed.Content>
+      </Feed.Event>
+    );
+  }
+
   ticketRow(ticket, index){
-    const title = (ticket.metadata.title) ? ticket.metadata.title : 'No name';
-    const rows = [];
+    const events = []
     const id = 'ticket_'+ticket._id+'_'+index;
-    if (index == 0) {
-      rows.push(<Table.Cell key={'title_'+id} rowSpan={ticket.actions.length}>{title}</Table.Cell>);
+    if (!ticket.actions) {
+      return ('No ticket actions');
```

Comment:
In rendering you generally always want to return a component. I'd just wrap this in a <span>